traitsui/qt4/ui_panel.py

```diff
@@ -1259,6 +1259,7 @@ class GroupEditor(Editor):
     def __init__(self, **traits):
         """ Initialise the object.
         """
+        HasPrivateTraits.__init__(self, **traits)
```

Comment:
For future reference, it'd be useful if you could add a comment here regd why `HasPrivateTraits.__init__` is being called instead of `Editor.__init__`. Linking to the issue is also a good idea.

Reply:
@aaronayres35 can you address this comment? Once you do, we can merge this PR

Reply:
Ah, yep sorry about the delay, this PR was lost in the shuffle.  I've added a comment now.